### DISCUSSION TITLE
Where can I download prism template pack for visual studio 2019? Seems like all the links only support 2022

### DISCUSSION THREAD

**Discussion by paulngugi00** [PrismLibrary/Prism#2799] [General]
I currently have vs 2019 installed and I still haven't raised enough cash to purchase vs 2022. Please can I get a download link from an old source?

"https://download.csdn.net/download/sinat_36203916/16292210?spm=1001.2101.3001.6650.8&utm_medium=distribute.pc_relevant.none-task-download-2%7Edefault%7EBlogCommendFromBaidu%7ERate-8-16292210-blog-124120014.pc_relevant_3mothn_strategy_recovery&depth_1-utm_source=distribute.pc_relevant.none-task-download-2%7Edefault%7EBlogCommendFromBaidu%7ERate-8-16292210-blog-124120014.pc_relevant_3mothn_strategy_recovery&utm_relevant_index=12"

I am unable to create a free account at the above site.

**Comment by brianlagunas** [comment]
VS2019 is not supported by the prism template pack
